ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -143,55 +157,116 @@ export class Send extends IronfishCommand {
       this.exit(1)
     }
 
-    if (fee == null) {
-      let suggestedFee = ''
-      try {
-        const response = await client.estimateFee({
-          fromAccountName: from,
-          receives: [
-            {
-              publicAddress: to,
-              amount: CurrencyUtils.encode(amount),
-              memo: memo,
-            },
-          ],
-        })
+    if (expiration !== undefined && expiration < 0) {
+      this.log('Expiration sequence must be non-negative')
+      this.exit(1)
+    }
+
+    let rawTransactionResponse: string
+    if (fee == null && feeRate == null) {
+      const feeRates = await client.estimateFeeRates()
+      const feeRateOptions: { value: number; name: string }[] = []
+
+      const createTransactionRequest: CreateTransactionRequest = {
+        sender: from,
+        receives: [
+          {
+            publicAddress: to,
+            amount: CurrencyUtils.encode(amount),
+            memo,
+            assetId,
+          },
+        ],
+        expiration: expiration,
+      }
+
+      const allPromises = []
+      if (feeRates.content.low) {
```

Comment:
It's not acceptable to test truthiness against numbers in javascript. If you want to know if this is returned, you must check this against `!= null`, or `!== undefined` which is the expect value you expect. As an engineer, this immediately looks like a bug to me because I now have to dive into the backend to see if this is expected or not and that saves me 5-8 minutes if you are just specific. Many cases where numbers are checked for truthiness are just bugs because the developer didn't think about what the value could be, or worse the possible value changed later.